### PR TITLE
Sample VBNET QBFC upgrade to 64 bit 

### PR DESCRIPTION
**Pull request by apatil0304**
JIRA: https://jira.intuit.com/browse/QBWG-135492
Reviewer : @ssadanandappa @5Anitha 

1. DataExtensions
2. DepositAdd
3. ExpenseReportByVendor
4. ItemInventoryMod
5. ItemReceipt
6. PriceLevels
7. SyncCustomerList

### CODE REVIEW

File: qbdt/vb.NET/QBFC/PriceLevels/PriceLevels.vbproj [hunk 1]
```diff
@@ -1,131 +1,158 @@
...
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningLevel>1</WarningLevel>
+    <NoWarn>42016,42017,42018,42019,42032,42353,42354,42355</NoWarn>
+    <DebugType>none</DebugType>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Interop.QBFC15">
+      <Name>Interop.QBFC15</Name>
+      <HintPath>..\..\..\..\..\..\..\..\..\..\Program Files\Common Files\Intuit\QuickBooks\Interop.QBFC15.dll</HintPath>
```
Comment (ssadanandappa) [inline qbdt/vb.NET/QBFC/PriceLevels/PriceLevels.vbproj:95]: Please hard code this to use C:\

File: qbdt/vb.NET/QBFC/PriceLevels/PriceLevels.vbproj [hunk 2]
```diff
@@ -1,131 +1,158 @@
...
-                    AssemblyName = "System.Windows.Forms"
-                />
-                <Reference
-                    Name = "System.XML"
-                    AssemblyName = "System.Xml"
-                />
-                <Reference
-                    Name = "Interop.QBFC14"
-                    AssemblyName = "Interop.QBFC14"
-                    HintPath = "..\..\..\..\..\..\..\..\..\..\Program Files (x86)\Common Files\Intuit\QuickBooks\Interop.QBFC14.dll"
```
Comment (ssadanandappa) [inline qbdt/vb.NET/QBFC/PriceLevels/PriceLevels.vbproj:83]: not needed .. please check

File: qbdt/vb.NET/QBFC/SyncCustomerList/SyncCustomerList.vbproj [hunk 3]
```diff
@@ -1,127 +1,160 @@
...
+    <WarningLevel>1</WarningLevel>
+    <NoWarn>42016,42017,42018,42019,42032,42353,42354,42355</NoWarn>
+    <DebugType>none</DebugType>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Interop.QBFC15, Version=15.0.0.1, Culture=neutral, PublicKeyToken=31d8aec643e18259, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <EmbedInteropTypes>True</EmbedInteropTypes>
+      <HintPath>..\..\..\..\..\..\..\..\Program Files\Common Files\Intuit\QuickBooks\Interop.QBFC15.dll</HintPath>
```
Comment (ssadanandappa) [inline qbdt/vb.NET/QBFC/SyncCustomerList/SyncCustomerList.vbproj:96]: Please use C:\

File: qbdt/vb.NET/QBFC/ItemReceipt/ItemReceipt.vbproj [hunk 4]
```diff
@@ -1,122 +1,157 @@
...
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningLevel>1</WarningLevel>
+    <NoWarn>42016,42017,42018,42019,42032,42353,42354,42355</NoWarn>
+    <DebugType>none</DebugType>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Interop.QBFC15">
+      <Name>Interop.QBFC15</Name>
+      <HintPath>..\..\..\..\..\..\..\..\..\..\Program Files\Common Files\Intuit\QuickBooks\Interop.QBFC15.dll</HintPath>
```
Comment (ssadanandappa) [inline qbdt/vb.NET/QBFC/ItemReceipt/ItemReceipt.vbproj:95]: Please use C:\

File: qbdt/vb.NET/QBFC/ItemInventoryMod/ItemInventoryMod.vbproj [hunk 5]
```diff
@@ -1,127 +1,184 @@
...
+    <Optimize>true</Optimize>
+    <WarningLevel>1</WarningLevel>
+    <NoWarn>42016,42017,42018,42019,42032,42353,42354,42355</NoWarn>
+    <PlatformTarget>x86</PlatformTarget>
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Interop.QBFC15">
+      <Name>Interop.QBFC15</Name>
+      <HintPath>..\..\..\..\..\..\..\..\..\..\Program Files\Common Files\Intuit\QuickBooks\Interop.QBFC15.dll</HintPath>
```
Comment (ssadanandappa) [inline qbdt/vb.NET/QBFC/ItemInventoryMod/ItemInventoryMod.vbproj:119]: Please use C:\

**Review by ssadanandappa — CHANGES_REQUESTED** [review]
Please use C:\ for the include paths in relative file and recheckin the zip files as well .. sorry if i missed it in the last review

File: qbdt/vb.NET/QBFC/SyncCustomerList/SyncCustomerList.vbproj (hunk 3 above)
Comment (apatil0304) [reply qbdt/vb.NET/QBFC/SyncCustomerList/SyncCustomerList.vbproj]: done, committed the changes

**Review by ssadanandappa — APPROVED** [review]

**Pull request merged**